### PR TITLE
Rex targetting

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

**Comment by lhellebr** [comment]
After rebase, this is broken by https://github.com/SatelliteQE/airgun/pull/1226

File: airgun/views/job_invocation.py [hunk 1]
```diff
@@ -54,6 +54,10 @@ class target_hosts_and_inputs(WizardStepView):
         ansible_roles_list = TextInput(id='ansible_roles_list')
         power_action = OUIASelect('OUIA-Generated-Select-single-34')
 
+        targetting_type = Select('OUIA-Generated-Select-single-2')
```
Comment (LadislavVasina1) [inline airgun/views/job_invocation.py:57]: I suggest using a locator parameter like this to generally target the correct element.

```suggestion
        targetting_type = Select(locator='//div[button[@aria-haspopup="listbox"]]')
```

File: airgun/views/job_invocation.py [hunk 2]
```diff
@@ -54,6 +54,10 @@ class target_hosts_and_inputs(WizardStepView):
...
+        # https://bugzilla.redhat.com/show_bug.cgi?id=2265127
+        targets = Select('OUIA-Generated-Select-typeaheadmulti-2')
```
Comment (LadislavVasina1) [inline airgun/views/job_invocation.py:59]: same here
```suggestion
        targets = Select(locator='//div[contains(@data-ouia-component-id,"OUIA-Generated-Select-typeaheadmulti-")]')
```

**Comment by LadislavVasina1** [comment]
> After rebase, this is broken by #1226

Should be fixed by this https://github.com/SatelliteQE/airgun/pull/1263

**Review by LadislavVasina1 — APPROVED** [review]
ACK

**Review by pondrejk — APPROVED** [review]

**Comment by lhellebr** [comment]
Can we merge?

**Pull request merged**